IFC4x3/Properties/t/TrackID_1QUq47Cd5EufPArQiLv0RG/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TrackID_1QUq47Cd5EufPArQiLv0RG" Name="TrackID" UniqueId="5a7b4107-3271-4ee2-964a-d5ab15e406d0" PrimaryDataType="IfcIdentifier" />
```

Comment:
Assuming that one track is in one system?

Reply:
"track system" is "track"